livre/eq/Client.cpp

```diff
@@ -57,7 +57,7 @@ struct Client::Impl
         }
 
         if( _applicationParameters.dataFileName.empty())
-            _applicationParameters.dataFileName = "mem:///#4096,4096,4096,32";
+            _applicationParameters.dataFileName = "mem:///#4096,4096,4096,40";
```

Comment:
In memory source ( in general blocks based volume rendering ) the blocks are padded, for sampling problems. Here 32 was the padded value. I wanted the blocks to be 32^3 exactly and 32 + 2 *4 = 40 is the block size now.